components/Accordion/Accordion.tsx

```diff
@@ -0,0 +1,56 @@
+import { useState } from 'react';
+import { MinusCircle, PlusCircle } from 'react-feather';
+
+import {
+  AccordionDetailsProps,
+  AccordionHeaderProps,
+  AccordionItemProps,
+  AccordionProps
+} from './Accordion.types';
+
+const AccordionItem: React.FC<AccordionItemProps> = ({ children }) => {
+  const [isExpanded, setIsExpanded] = useState<boolean>(false);
+
+  const onToggle = () => {
+    setIsExpanded(!isExpanded);
+  };
+
+  return (
+    <div className={'border-b py-3 md:pb-7'}>
+      <div
```

Comment:
this is also a button as a semantic mean. you can check this site for semantic and accessible pattern for accordion component. https://www.w3.org/WAI/ARIA/apg/example-index/accordion/accordion.html